cfme/containers/provider/__init__.py

```diff
@@ -331,7 +331,9 @@ def __init__(self, obj, polarion_id):
         self.polarion_id = polarion_id
 
     def pretty_id(self):
-        return '{} ({})'.format(self.obj.__name__, self.polarion_id)
+        name = (self.obj.__name__ if hasattr(self.obj, '__name__')
```

Comment:
Optionally keep it as a single statement as well, and stick `getattr()` into the `format()` call.